app/src/main/java/com/devmob/alaya/ui/screen/professionalHome/ProfessionalHomeViewModel.kt

```diff
@@ -23,4 +27,14 @@ class ProfessionalHomeViewModel : ViewModel() {
         users = UsersProvider.users
     }
 
+    private fun updateGreetingMessage() {
+        val calendar = Calendar.getInstance()
+        val hourOfDay = calendar.get(Calendar.HOUR_OF_DAY)
+        greetingMessage = when (hourOfDay) {
+            in 0..11 -> "Buenos días"
```

Comment:
Lo mismo que en el paciente:
a la 1 de la mañana diría "Buenos días"? Seria mejor lo siguiente?
`in 5..11 -> "Buenos días"`